_docs/concepts/auth.md

```diff
@@ -0,0 +1,138 @@
+---
+category: Concepts
+title: Auth
+overview: Architectural deep-dive into the design of Auth, which provides the secure communication channel and strong identity for Istio.
+              
+parent: Network and Auth
+order: 20
+
+bodyclass: docs
+layout: docs
+type: markdown
+---
+
+The page explains Auth's role and general architecture.
+
+
+## Istio Auth Concept
+
+*Status: Draft*
+
+*Date: April, 2017*
+
+
+[TOC]
+
+
+### Overview {#overview}
+
+Istio Auth aims at enhancing the security of microservices and their communication without requiring service code changes. It is responsible for:
+
+
+
+*   Presenting a strong identity that represents the role of the service to enable interoperability across clusters and clouds.
+*   Securing the service to service communication and the end-user to service communication.
+*   Providing a key management system to automate key/cert generation, distribution, rotation, and revocation.
+*   Upcoming features:
+    *   fine-grained authorization and auditing to control and monitor who accesses your services, apis, or resources, 
+    *   powerful authorization mechanisms: [ABAC](https://en.wikipedia.org/wiki/Attribute-Based_Access_Control), [RBAC](https://en.wikipedia.org/wiki/Role-based_access_control), Authorization hooks.
+
+### Architecture {#architecture}
+
+Figure 1 shows the Istio Auth architecture, which includes three important components: identity, key management, and communication security. This diagram describes how Istio Auth is used to secure the service-to-service communication between service A running as the service account "foo" and service B running as the service account "bar".
+
+<img style="display:block;width:60%;margin:auto;" src="./img/auth/auth.svg"
+alt="Flow of traffic." />
+
+### Components {#components}
+
+#### Identity {#identity}
+
+Istio Auth uses [Kubernetes service accounts](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/) to identify who runs the service:
+
+
+
+*   Service account is **the identity (or role) the workload runs as**, which represents the privilege of the workload. For the system requiring strong security, the amount of privilege for a workload should not be identified by a random string (i.e., service name, label, etc), or by the binary that is deployed.
```

Comment:
"For the system requiring strong security, the amount of privilege for a workload should not be identified by a random string (i.e., service name, label, etc), or by the binary that is deployed."

I'm a) not sure if this fits here (the following "for example..." is an example of "the identity a workload runs as...", plus it seems to be going from "here is what Istio provides" to "here is something that should be done"), and b)amn't entirely sure what it means. Does it mean that using a service account )as Istio Auth does) is a *better alternative* to identifying a workload's privileges by a particular string or deployed binary?